Examples/Widgets/CovidByLocation/src/index.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { html, LitElement, customElement, css, internalProperty, property } from "lit-element";
+import "./components/App";
+/**
+ * Please give your widget a unique name. We recommend using prefix to identify the author and help avoid naming conflict. e.g. "2ring-timer-widget"
+ */
+@customElement("covid-by-location")
+export default class CovidByLocation extends LitElement {
+  @internalProperty() private contacts: string[] = [];
+  @property({ type: String }) selectedCountyState = "Santa Clara County, CA";
+  /**
+   * Property: apiKey
+   * Access API Key: Covid Act Now Website
+   * https://apidocs.covidactnow.org/access
+   */
+  @property({ type: String, reflect: true, attribute: "api-key" }) apiKey = "1318b408f01c4aa3b5f79dedc6c90848";
```

Comment:
we should remove this api key, right?

Reply:
haha yes, I keep removing, but then putting it back to test :) I'll remove it.